coursebuilder/modules/explorer/_static/css/style.css

```diff
@@ -3594,7 +3594,7 @@ footer .footer-bottom .footer-social-icons a { color: black; }
 
 .course-card { position: relative; }
 
-.course-card img { width: 100%; }
+.course-card img { width: 100%; max-height: 400px; }
```

Comment:
@andela-iakande should this be something set in @media screen?

Reply:
@andela-iakande Also how do you take care of inline styling set in the editor?

Reply:
@chegejames  the max-height was not placed in the @media screen it is the way gulp compiled into CSS styles that make it appear that way. About the inline styling, please can you expatiate because I did not use inline styles here

Reply:
@andela-iakande https://cforafrica-mooc-test.appspot.com/test/unit?unit=1&lesson=3 How do you do it without losing aspect ratio? Why 400px as max-height? Should it be max-width:100% as opposed to setting the width?

Reply:
@andela-iakande for inline styling, https://gmkr.io/s/59db436bfecd64d446e48991/0 element.style {
    width: 500px;
    height: 520px;
} will be inline.

Reply:
@adamoxford1 suggested we set a max height. He stated that Width is scaling to 100%, but height seems to be arbitrary? Leads to images being stretched in most views. https://trello.com/c/FfXE8H7Q/112-feedback-images-arent-scaling-correctly-in-responsive-views. I assigned 400px height so as to allow card images uniform and not overstretch in some exceptional cases and I discovered it worked across.

Reply:
@andela-iakande have a look at that https://cforafrica-mooc-test.appspot.com/test/unit?unit=1&lesson=3

Reply:
@chegejames when I added inline styles of width: 500px; and
height: 520px; to the course card. it did not scale well for more than one image in a lesson

Reply:
@andela-iakande https://cforafrica-mooc-test.appspot.com/test/unit?unit=1&lesson=3  this is what I mean by setting both max-height:400px and width:100%